### PR TITLE
Workshop for arma

### PR DESCRIPTION
**Pull request by JiaZhou-PU**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)

Creat workshop presentation and exercises for ARMA

##### What are the significant changes in functionality due to this change request?

Creat workshop presentation for ARMA with a case study in Wind speed and load profile:
EX1: Train arma model and generate the synthetic sample for wind speed
EX2: Train arma and Fourier model and generate the synthetic sample for load profile
EX3: Pickle the trained arma model with a random number seed 
EX4: Unpickle the pk file and generate samples again and compare the samples in EX3
EX5: Use synthetic histories from load and wind speed to ensemble an economic model

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [ ] 1. Review all computer code.
- [ ] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [ ] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [ ] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [ ] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [ ] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [ ] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [ ] 8. If an analytic test is changed/added is the the analytic documentation updated/added?
- [ ] 9. If any test used as a basis for documentation examples (currently found in `raven/tests/framework/user_guide` and `raven/docs/workshop`) have been changed, the associated documentation must be reviewed and assured the text matches the example.

### CODE REVIEW

**Comment by JiaZhou-PU** [comment]
I need some final touch up, but ready for review... @PaulTalbot-INL

**Comment by alfoa** [comment]
Can we review/merge this? What do you think? I would like to avoid to "waste" this effort.
Andrea

**Comment by PaulTalbot-INL** [comment]
Just that one change, and we should definitely preserve this, I agree.

**Comment by JiaZhou-PU** [comment]
Is the workshop material have already changed? I remember there is an auto filling part. @alfoa @PaulTalbot-INL

**Comment by PaulTalbot-INL** [comment]
Part of the workshop has automated exercise filling for testing, but it's not ubiquitous yet

File: doc/workshop/ARMA/inputs/1_arma.xml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>/Users/zhouj2/projects/raven/doc/workshop/ARMA/Inputs.ARMA1</name>
```
[the remark this answers is not in the document]
Comment (JiaZhou-PU) [reply doc/workshop/ARMA/inputs/1_arma.xml]: Addressed

**Comment by PaulTalbot-INL** [comment]
@alfoa We don't have a current policy on testability of workshop exercises, officially, right? We want to extend the exercise filling to all the exercises, but have not yet.

Is this okay to merge without it, then, and we'll add the exercise filling to the rest when we expand coverage?

**Comment by alfoa** [comment]
Yes it is in my opinion.

Please go ahead.
Andrea

**Comment by alfoa** [comment]
I close this since another PR has been opened

**Pull request closed without merging**